wiki/People/Performance_Points_Committee/en.md

```diff
@@ -47,12 +47,15 @@ The osu! panel was first announced to the public through the [November 2021 *Per
 
 ### osu!taiko
 
-The osu!taiko panel was formed on 1 May 2022 in response to [a community survey regarding performance points and star rating in osu!taiko and osu!mania](https://osu.ppy.sh/home/news/2022-01-14-taiko-mania-pp-sr-survey) held earlier that year, as well as [major changes to the star rating algorithm](https://docs.google.com/document/d/1Z5GC4DMqOVzeIERMSK3qpQaqjq-sVnhbuoxAwy9qxDs/edit).
+The osu!taiko panel was formed on 1 May 2022 in response to [a community survey regarding performance points and star rating in osu!taiko and osu!mania](https://osu.ppy.sh/home/news/2022-01-14-taiko-mania-pp-sr-survey) held earlier that year, as well as [major changes to the star rating algorithm](https://docs.google.com/document/d/1Z5GC4DMqOVzeIERMSK3qpQaqjq-sVnhbuoxAwy9qxDs/edit). In December 2024, the osu!taiko panel was once again updated via an application process alongside [major changes to the star rating algorithm](https://docs.google.com/document/d/1-k96U-3-77CSyWNbXxa3cf2AXeOIxVh8yoXowlIYc5g/edit?usp=sharing).
```

Comment:
mentioning this might be all well and good, but we've never logged any of the other changes to the committee

Reply:
Fair call, this one just depends on the fact that it was updated in response to major changes occurring, rather than members picked up along the way

Reply:
would move this to some trivia/history heading

Reply:
also what is this google document? does it describe future or already-implemented changes (that are on github/a news post)? considering 2024 is almost over it seems to me that it's related to the [last batch of updates](https://osu.ppy.sh/home/news/2024-10-28-performance-points-star-rating-updates) but doesn't link it

Reply:
By the end of 2024, probably this will turn into 2025 future implementation, linked by the latest pr pending review.